projects/dashboards/src/widgets/container/layout/widget-theme.ts

```diff
@@ -0,0 +1,4 @@
+export enum WidgetTheme {
+  Dark = 'dark',
```

Comment:
Should we call it Gray, instead of Dark? Dark theme gives me a Dark mode feel

Reply:
Done.